partner_credit_limit/i18n/es.po

```diff
@@ -100,3 +81,56 @@ msgstr "Valor incorrecto %s para los días de gracia de pago: el valor debe esta
 #: model:ir.model,name:partner_credit_limit.model_sale_order
 msgid "Quotation"
 msgstr "Presupuesto"
+
+#. module: partner_credit_limit
+#: code:addons/partner_credit_limit/models/partner.py:107
+#, python-format
+msgid "Please: \n"
+""
+msgstr "Por favor: \n"
+""
+
+#. module: partner_credit_limit
+#: code:addons/partner_credit_limit/models/sale.py:26
+#: model:ir.model,name:partner_credit_limit.model_sale_order
```

Comment:
@hugho-ad @CarmenMiranda @luisg123v 

I think this is the translate in the issue, this line is twice here and line 81

Reply:
Yes, just minutes ago I saw this, I'm doing a DB to check it with the overwrite included

Reply:
Yes, that was the issue, I already delete this line :smile: